### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by alax99**
#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [![Badge shield](https://circleci.com/gh/zixuan75/html_project.svg?style=shield&circle-token=w)](https://circleci.com/gh/zixuan75/html_project.svg?style=shield&circle-token=w)
 [![Build Status](https://travis-ci.org/zixuan75/html_project.svg?branch=master)](https://travis-ci.org/zixuan75/html_project)
 # html_project
-A HTML and JavaScript program that runs browser Firefox and Chrome (access to Sadira).
+A HTML and JavaScript program that runs browser Firefox and Chrome (access to Sadira and Java programs).
```
Comment (zixuan75) [inline README.md]: Why java program? Oh no

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [![Badge shield](https://circleci.com/gh/zixuan75/html_project.svg?style=shield&circle-token=w)](https://circleci.com/gh/zixuan75/html_project.svg?style=shield&circle-token=w)
```
Comment (zixuan75) [inline README.md]: Shiled tokens

Comment (zixuan75) [reply README.md]: Those is a shield.

**Comment by zixuan75** [comment]
Sadira

**Comment by alax99** [comment]
foid

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 [![Build Status](https://travis-ci.org/zixuan75/html_project.svg?branch=master)](https://travis-ci.org/zixuan75/html_project)
 # html_project
-A HTML and JavaScript program that runs browser Firefox and Chrome (access to Sadira).
+A HTML and JavaScript program that runs browser Firefox and Chrome (access to Sadira and Java programs).
 
 While some features is supported for Firefox and Chrome, but it is not supported for Internet Explorer, because it is just not and that does not meet the browser Internet Explorer, so that is why some features is not supported for Internet Explorer. The **repository** tag is not supported for Internet Explorer (also you need to clone this repository to access your **repository** tag by running this command: `git clone https://github.com/zixuan75/html_project.git`).
```
Comment (zixuan75) [inline README.md]: Fou no.

**Comment by zixuan75** [comment]
#49

**Pull request merged**